src/main/java/net/starlark/java/eval/Dict.java

```diff
@@ -234,16 +234,18 @@ public Object pop(Object key, Object defaultValue, StarlarkThread thread) throws
               + "<code>popitem()</code> is useful to destructively iterate over a dictionary, "
```

Comment:
Let's change this comment to say "the first", not "an arbitrary", as that's what the spec requires:
 https://github.com/bazelbuild/starlark/blob/master/spec.md#dict%C2%B7popitem